### PR TITLE
Init app template

### PR DESCRIPTION
**Pull request by egorkozhadei**
- Create webpack config
- Add static files and templates
- Configure ESLint and Prettier
- Add TailwindCSS and PostCSS configurations
- Add simple TailwindCSS template
- Add pre-commit hook with Husky

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run prettier:check
+npm run lint
+npm run tscheck
```
Comment (Gudashev) [inline .husky/pre-commit]: add empty line in the end of the file

Comment (egorkozhadei) [reply .husky/pre-commit]: Thanks! Resolved.

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
+.app {
```
Comment (Gudashev) [inline src/App.css:1]: you use tailwind css

File: assets/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <!--
```
Comment (Gudashev) [inline assets/index.html:13]: remove comments

File: src/App.css (hunk 2 above)
Comment (Gudashev) [reply src/App.css:1]: so delete css styles  and connect tailwind

**Review by Gudashev — APPROVED** [review]
merge after all comments have been corrected

**Pull request merged**